contrib/mypy/tests/python/pants_test/contrib/mypy/tasks/BUILD

```diff
@@ -9,4 +9,14 @@ python_tests(
     'tests/python/pants_test:interpreter_selection_utils',
   ],
   tags={'integration'},
-)
\ No newline at end of file
+)
+
+python_tests(
+  name='mypy',
+  source = 'test_mypy.py',
+  dependencies=[
+    'tests/python/pants_test:task_test_base',
+    'contrib/mypy/src/python/pants/contrib/mypy/tasks',
+  ],
+  timeout = 6000,
```

Comment:
This isn't a good timeout: should lower it to ~50% more time than the test takes to run on average.

Reply:
Good catch, that's a remnant from debugging.